src/vs/vscode.d.ts

```diff
@@ -2972,6 +2972,37 @@ declare namespace vscode {
 	}
 
 	/**
+	 * An individual terminal instance within the integrated terminal.
+	 */
+	export interface Terminal {
+
```

Comment:
I added that and then reverted it, it never changes currently so they should be able to keep their old reference? Figured this would enable more flexibility later on if name becomes more dynamic (`setName`, `getName` or something.)